aten/src/ATen/TensorUtils.cpp

```diff
@@ -215,4 +215,23 @@ void * maybe_data_ptr(const Tensor& tensor) {
 void * maybe_data_ptr(const TensorArg& tensor) {
   return tensor->defined() ? (void *)tensor->data_ptr() : nullptr;
 }
+
+bool geometry_is_contiguous(IntList sizes, IntList strides) {
```

Comment:
FYI, we're caching `is_contiguous` on `Tensor` now, so that'll be better use than this function. Might be worth a comment here.